src/Core/Parser/SyntaxTree/Expression/MathExpressionNode.php

```diff
@@ -26,7 +26,7 @@ class MathExpressionNode extends AbstractExpressionNode
 		(
 			{                                # Start of shorthand syntax
 				(?:                          # Math expression is composed of...
-					[a-zA-Z0-9\.]+[\s]?[*+\^\/\%\-]{1}[\s]?[a-zA-Z0-9\.]+   # Various math expressions left and right sides with any spaces
+					[a-zA-Z0-9\.]+(?:[\s]?[*+\^\/\%\-]{1}[\s]?[a-zA-Z0-9\.]+)+   # Various math expressions left and right sides with any spaces
```

Comment:
Thanks for the explanation and examples! Please add these to the commit message and adjust the topic so it explains camelCase isn't the only bug fixed - alternatively, split this into two commits which fix the two bugs separately.